libraries/core-react/src/components/Snackbar/Snackbar.tokens.ts

```diff
@@ -17,46 +17,48 @@ const {
     },
   },
   spacings: {
-    comfortable: { medium: spacingMedium },
+    comfortable: { medium: spacingMedium, x_large: spacingXLarge },
   },
   elevation: { overlay: boxShadow },
   clickbounds: { default__base: clickbounds },
+  shape: {
+    button: { borderRadius: radius },
+  },
 } = tokens
 
-type Snackbar = {
-  background: string
+type Snackbar = ComponentToken & {
   boxShadow: string
```

Comment:
We don't want to "mutate" `ComponentToken` like this. Suggest using (a styled)`Paper` component inside `Snackbar` with the correct `elevation` set.